src/views/SelectProduct.vue

```diff
@@ -157,14 +157,25 @@ export default defineComponent({
 </script>
 
 <style scoped>
-.filters {
-  border-right: 1px solid var(--ion-color-medium);
-}
 .section-grid {
   grid-template-columns: repeat(auto-fill, 200px);
 }
 
 @media (min-width: 991px) {
+  .find {
+    grid: "search  filters"
+          "main    main"
+          / 440px 400px;        
```

Comment:
I don't think we need to set any explicit values on columns right now. By default I think it should split the space in half. We can manage width of search and select on the elements themselves.

Reply:
I have removed explicit values on columns.